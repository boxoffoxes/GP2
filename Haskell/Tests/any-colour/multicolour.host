[
	(n1, empty)
	(n2, empty # red)
	(n3, empty # green)
	(n4, empty # blue)
	(n5, empty # grey)
|
	(e1, n1, n1, empty)
	(e2, n2, n2, empty)
	(e3, n3, n3, empty)
	(e4, n4, n4, empty)
	(e5, n5, n5, empty)
]
