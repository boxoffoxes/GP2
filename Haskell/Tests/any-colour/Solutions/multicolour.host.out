[
 (n1, empty # grey)
 (n2, empty # blue)
 (n3, empty # green)
 (n4, empty # red)
 (n5, empty)
|
]
